source/microbit/help.c

```diff
@@ -132,7 +132,7 @@ STATIC const mp_doc_t help_table_instances[] = {
     {&microbit_pin_is_touched_obj, "If pin is_touched() on micro:bit, return True. If nothing is touching the pin,\nreturn False.\n"},
     // I2C
     {&microbit_i2c_obj, "Communicate with one or more named devices connected to micro:bit. Each named\ndevice has an 'address', communicates using I2C, and connects to the I/O pins.\n"},
-    {&microbit_i2c_set_frequency_obj, "Use set_frequency(f) to set the I2C bus frequency.\n"},
+    {&microbit_i2c_init_obj, "Use init() to set up communication with a clock frequench of 100 KHz.\nOverride the default for 'frequency' in Hz.\n"},
```

Comment:
I quite like the spi init description `Use init() to set up communication. Override the defaults for baudrate, mode,\nSCLK, MOSI and MISO. The default connections are pin13 for SCLK, pin15 for\nMOSI and pin14 for MISO.\n`, makes it clear that you can call without arguments or overwrite defaults. So maybe something slightly shorter like?:

`Use init() to set up communication. Override the 100KHz default for 'frequency' with a value in Hz."`